cg/cli/post_process/post_process.py

```diff
@@ -0,0 +1,32 @@
+"""CLI commands to post-process a sequencing run."""
+
+import logging
+
+import click
+
+from cg.cli.post_process.utils import get_post_processing_service_from_run_name
+from cg.cli.utils import CLICK_CONTEXT_SETTINGS
+from cg.models.cg_config import CGConfig
+from cg.services.post_processing.abstract_classes import PostProcessingService
+
+LOG = logging.getLogger(__name__)
+
+
+@click.group(name="post-process", context_settings=CLICK_CONTEXT_SETTINGS)
+def post_process_group():
+    """Add sequencing data to the databases."""
```

Comment:
```suggestion
    """Post-processes sequencing run from the following run instruments: PacBio."""
```
extend when implemented more run instruments